rest-servlet/src/main/java/org/hawkular/inventory/rest/RestResourceTypes.java

```diff
@@ -112,8 +113,9 @@ public ResourceType get(@PathParam("tenantId") String tenantId,
     public Set<Resource> getResources(@PathParam("tenantId") String tenantId,
                                       @PathParam("resourceTypeId") String resourceTypeId) {
 
-        return inventory.tenants().get(tenantId).resourceTypes().get(resourceTypeId)
-                .resources().getAll().entities();
+        ResourceTypes.Single single = inventory.tenants().get(tenantId).resourceTypes().get(resourceTypeId);
```

Comment:
ok, now I see your point. I thought, you were talking about other entity types in the Java API. 

It's pretty much everywhere in the REST:
/inventory/no-tenant/test/{resources|metrics}
/inventory/test/no-env/{resources|metrics}
/inventory/no-tenant/no-env/resources/no-resource/metrics
/inventory/no-tenant/metricTypes
/inventory/no-tenant/resourceTypes/no-resType/metricTypes
...
it all returns [] 200 :/